release.sh

```diff
@@ -572,7 +572,7 @@ fi
 package=
 manual_changelog=
 changelog=
-changelog_markup=plain
+changelog_markup=text
```

Comment:
I checked it's safe to make that change as changelog_markup is only used below (beside tested for being equal to "markdown" or not)

Reply:
quote that while you're there
```suggestion
changelog_markup="text"
```

Reply:
was keeping it consistent with what it was (and some other variables like  repository_type) but sure